service/src/test/java/uk/gov/hmcts/reform/fpl/service/email/NotificationServiceTest.java

```diff
@@ -17,15 +20,18 @@
 import static uk.gov.hmcts.reform.fpl.NotifyTemplates.PARTY_ADDED_TO_CASE_BY_EMAIL_NOTIFICATION_TEMPLATE;
 
 @ExtendWith(SpringExtension.class)
+@ContextConfiguration(classes = {NotificationService.class, NotificationClient.class, JacksonAutoConfiguration.class})
+@TestPropertySource(properties = {"fpl.env=localhost" })
 public class NotificationServiceTest {
     public static final String TEST_RECIPIENT_EMAIL = "test@example.com";
     public static final String REFERENCE = "12345L";
     public static final String TEMPLATE_ID = PARTY_ADDED_TO_CASE_BY_EMAIL_NOTIFICATION_TEMPLATE;
+    private static final String CASE_REFERENCE_WITH_ENVIRONMENT = "localhost/" + REFERENCE;
```

Comment:
could be


@TestPropertySource(properties = {"fpl.env=" + ENV})
public class NotificationServiceTest {
  ....
    static final String ENV = "TEST_ENV";
    private static final String CASE_REFERENCE_WITH_ENVIRONMENT = String.format("%s/%s", ENV, REFERENCE);

Reply:
done